### ISSUE TITLE
Pagination

### ISSUE THREAD

**Issue by infiniteluke** [infiniteluke/next-static#3]
Sooner rather than latter, showing all posts on the home page will be a bad idea. Paging can help that.

Should include a config option for `pageSize` to control how many posts are in each page. Url should look like `mysite.com/?page=2`, and aliasing page 1 to the home page.

**Comment by jacobpdq** [comment]
At this point, I believe the best method is to use graphQL and pre-parse the large post list into JSON fragments. There's no better way to go here.

I'm assuming json-graphql-server is the way to go.

**Comment by infiniteluke** [comment]
I don't think that is necessary. I'll try to find some time this week to get pagination implemented as an example for you.
